### ISSUE TITLE
new bug: geolocate in frontend>editvenue doesn't work anymore (with apikey)

### ISSUE THREAD

**Issue by jojo12** [Heklaterriol/JEM-Project#211]
I've seen you changed something e.g. buttons in backend editvenue. I don't know what you changed in frontend.
But there the map is away, I can't use the "find with JEM-.."anymore. If I write an address into the geocomplete field, I get the map from the whole world! In backend it works.

**Comment by Heklaterriol** [comment]
Ther was no default top level domain ending "com", if not set in JEM settings

**Comment by jojo12** [comment]
doesn't work in editvenue!

**Comment by jojo12** [comment]
even without api-key and even with link to map

**Comment by Heklaterriol** [comment]
Strange – work fine here: Can you send me an URL to check (via PM in the JEM Forum)

**Comment by jojo12** [comment]
it looks like the map is taking (0 North, 0 East)

**Comment by jojo12** [comment]
in backend and editvenue there is a map and it works, but then in venue details and eventdetails+venue is the world map.

**Comment by jojo12** [comment]
yes I tried, it works without api and with api from frontend and backend.
when api is filled in, then in backend and frontend editvenue, there is a hybrid map and I am able to move markers to the right positition and to take it back to JEM!
and frontend shows always the right map!    puuh! Thanks a lot!

but I got the following:
1062 Duplicate entry 'defaultCountry' for key 'PRIMARY'

Warning: Undefined property: stdClass::$defaultCountry in C:\xampp\htdocs\joomla_42\components\com_jem\models\editvenue.php on line 121

Warning: Undefined property: stdClass::$defaultCountry in C:\xampp\htdocs\joomla_42\administrator\components\com_jem\models\venue.php on line 195

**Comment by jojo12** [comment]
something is strange! when I fill a new venue, I get always coordinates 0 / 0. 
Only if I move the marker, I get right coordinates

**Comment by Heklaterriol** [comment]
> but I got the following: 1062 Duplicate entry 'defaultCountry' for key 'PRIMARY'
> 
> Warning: Undefined property: stdClass::$defaultCountry in C:\xampp\htdocs\joomla_42\components\com_jem\models\editvenue.php on line 121
> 
> Warning: Undefined property: stdClass::$defaultCountry in C:\xampp\htdocs\joomla_42\administrator\components\com_jem\models\venue.php on line 195

Do you have a default country set in > Settings > Global Parameters?
And if not and you choose one – does is change the warnings?

**Comment by jojo12** [comment]
when I don't have a country, I get:
Speichern fehlgeschlagen. Fehler: Sind folgende Felder ausgefüllt: Strasse, Stadt, Bundesland, PLZ
But it says nothing about "country"!

**Comment by jojo12** [comment]
Don't change it I can live with that!
